{
  "cve_id": "CVE-2023-38499",
  "timestamp": "2025-01-14 16:58:36",
  "phases": {
    "primary": {
      "completed": true,
      "timestamp": "2025-01-10 10:39:53"
    },
    "secondary": {
      "completed": true,
      "timestamp": "2025-01-11 20:55:27"
    },
    "extraction": {
      "completed": true,
      "timestamp": "2025-01-12 06:22:15"
    }
  },
  "urls": [
    {
      "url": "https://github.com/TYPO3/typo3/commit/702e2debd4b28f9cdb540544565fe6a8627ccb6a",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://github.com/TYPO3/typo3/security/advisories/GHSA-jq6g-4v5m-wm9r",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    },
    {
      "url": "https://typo3.org/security/advisory/typo3-core-sa-2023-003",
      "status": "success",
      "reason": null,
      "timestamp": "2025-01-14 11:33:06"
    }
  ]
}